File: program/swap-two-numbers/swapTwoNumbers.scala
object Main extends App {
    var a = scala.io.StdIn.readInt()
    var b = scala.io.StdIn.readInt()
    
    println("Before Swapping a = " +a+ " b =" +b)

    a = a + b
    b = a - b
    a = a - b

    println("After Swapping a = " +a+ " b =" +b)
}

File: program/program/swap-two-numbers/swap_two_numbers.php
<?php
	$a = 10;
	$b = 20;

	echo "Before a: ".$a.", b: ".$b."\n";

	$a = $a + $b;
	$b = $a - $b;
	$a = $a - $b;

	echo "After  a: ".$a.", b: ".$b;
?>
